### PR TITLE
Generate useful repo README file

### PR DESCRIPTION
**Pull request by michaeljguarino**
## Summary

This could potentially help onboard users quicker to understanding their repo. Refactored to just pull from our docs github so we can easily iterate on it as well.

## Test Plan
executed `plural readme` to confirm it generates appropriately

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Comment by DavidSpek** [comment]
What happens if the file already exists?

**Comment by michaeljguarino** [comment]
It would just overwrite it currently, but the only paths it executes currently is init and shell setup, so really shouldn't overwrite ever

File: pkg/server/setup.go [hunk 1]
```diff
@@ -118,6 +119,7 @@ func setupCli(c *gin.Context) error {
 	if err := crypto.CreateKeyFingerprintFile(); err != nil {
 		return err
 	}
+	_ = wkspace.DownloadReadme()
```
Comment (zreigz) [inline pkg/server/setup.go:122]: why did you hide the error?

**Review by DavidSpek — APPROVED** [review]
LGTM

**Review by swoodward90 — APPROVED** [review]

File: pkg/server/setup.go (hunk 1 above)
Comment (michaeljguarino) [reply pkg/server/setup.go:122]: it's on a critical path and I don't want init to fail if it fails (eg the user still sets up but maybe doesn't have a readme in their repo, no big deal)

**Pull request merged**